package-lock.json

```diff
@@ -5167,6 +5167,12 @@
           "resolved": "https://registry.npmjs.org/regexp-tree/-/regexp-tree-0.1.13.tgz",
           "integrity": "sha512-hwdV/GQY5F8ReLZWO+W1SRoN5YfpOKY6852+tBFcma72DKBIcHjPRIlIvQN35bCOljuAfP2G2iB0FC/w236mUw==",
           "dev": true
+        },
+        "semver": {
```

Comment:
Can you revert your `package-lock.json` changes? They are unnecessary for the TS migration.

Reply:
Made the revert :)